tests-functional/clients/signals.py

```diff
@@ -3,32 +3,63 @@
 import time
 
 import websocket
+import os
+from pathlib import Path
+from constants import SIGNALS_DIR, LOG_SIGNALS_TO_FILE
+from datetime import datetime
+from enum import Enum
 
+class SignalType(Enum):
+    MESSAGES_NEW = "messages.new"
+    MESSAGE_DELIVERED = "message.delivered"
+    NODE_READY = "node.ready"
+    NODE_STARTED = "node.started"
+    NODE_LOGIN = "node.login"
+    MEDIASERVER_STARTED = "mediaserver.started"
+    WALLET_SUGGESTED_ROUTES = "wallet.suggested.routes"
+    WALLET_ROUTER_SIGN_TRANSACTIONS = "wallet.router.sign-transactions"
+    WALLET_ROUTER_SENDING_TRANSACTIONS_STARTED = "wallet.router.sending-transactions-started"
+    WALLET_TRANSACTION_STATUS_CHANGED = "wallet.transaction.status-changed"
+    WALLET_ROUTER_TRANSACTIONS_SENT = "wallet.router.transactions-sent"
```

Comment:
Please, let's use the PEP8 naming convention everywhere - variables should be lowercase with underscores. Otherwise looks good, thanks!